### ISSUE TITLE
Better description of DevEdition on our websites

### ISSUE THREAD

**Issue by violasong** [firefox-devtools/ux#93]
Mainly concerning the [landing page](https://www.mozilla.org/en-US/firefox/developer/), but also [download pages](https://www.mozilla.org/en-US/firefox/channel/desktop/).

Incorporate info from [the docs](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Developer_Edition), and include up-to-date benefits such as the specific features that are DevEdition-only (not yet in Beta).

**Comment by violasong** [comment]
<img width="918" alt="image" src="https://user-images.githubusercontent.com/55994/64914154-4cce3680-d702-11e9-82e0-7132949b9a8a.png">

This is confusing. What's the actual name of this product?

**Comment by fvsch** [comment]
>  It's similar to Beta, except that installing Beta is potentially dangerous as it will override users' GA Firefox with a new profile

Wasn't that changed a few months ago? Now there's one profile per install (or channel?).

"Quantum" seems out indeed. The correct name seems to be "Firefox Developer Edition", though the wordmark contradicts it a bit since it spells out "Firefox Browser [Developer]". Maybe the title should say "Firefox Browser: Developer Edition"?

Current page with "Firefox Developer Edition" title:

![Screen Shot 2019-09-16 at 16 59 33](https://user-images.githubusercontent.com/243601/64969086-88eccd00-d8a3-11e9-842a-f0c3963dd3cf.png)

The https://mozilla.design/firefox/logos-usage/ page still shows the Firefox 69 wordmark:

![FF_Developer_Edition__logo-and-wordmark-horizontal](https://user-images.githubusercontent.com/243601/64989761-86539d00-d8ce-11e9-816a-823ff2be3f7b.png)

**Comment by violasong** [comment]
>>   It's similar to Beta, except that installing Beta is potentially dangerous as it will override users' GA Firefox with a new profile

>Wasn't that changed a few months ago? Now there's one profile per install (or channel?).

Ah, whoops, I'll update this. The problem is actually that it shares the same path as GA. According to @digitarald: 

> biggest downside is that installing Beta will override users GA Firefox, as they are using the same paths (aka firefox.exe). Launching Beta will then create a new profile (as profiles are now locked to channels). I am not sure how many users are able to recover from that and can find their data again.

This page would also need to be updated
https://www.mozilla.org/en-US/firefox/70.0a2/whatsnew/

**Comment by martinbalfanz** [comment]
@violasong it would be great if we could also add the accessibility panel to the page. it's missing from the tool overview at the bottom.

**Comment by violasong** [comment]
Good point, I'd just have to make the icon We should probably remove two of the tools to keep that section at 3 rows. (Memory and Storage maybe.) Ah, and I see the Visual Editing section links to a blog post from 2015. ...Maybe this section could be part of a milestone 2 :)

**Comment by violasong** [comment]
M1 of updates:

1. Replace contents of `<h1 class="intro-title">` with `Firefox Developer Edition`

2. Replace "Faster Performance" section with this text:
```
<p class="c-subtitle">The browser made for developers</p>
<p>
All the latest developer tools in beta, plus <strong>experimental features</strong> like the Multi-line Console Editor and WebSocket Inspector.
</p>

<p>
A <strong>separate profile</strong> so you can easily run it alongside Release or Beta Firefox.
</p>

<p>
Preferences <strong>tailored for web developers</strong>: Browser and remote debugging are enabled by default, as well as the dark theme and developer toolbar button.
</p>
```



Should look like this:

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/55994/66519550-5eee8b00-ea9c-11e9-87bb-66894e485662.png">

**Comment by violasong** [comment]
Next steps: 
- Update screenshots
- Add a release notes link
- Look into localization

**Comment by craigcook** [comment]
Updates are live.

https://www.mozilla.org/en-US/firefox/developer/
https://www.mozilla.org/en-US/firefox/70.0a2/whatsnew/
https://www.mozilla.org/en-US/firefox/70.0a2/firstrun/

The new content is only in English and other locales continue to get the old content. We can expose the new strings for translation but since it sounds like there are more changes forthcoming we should wait until everything is ready and localize all at once.

**Comment by violasong** [comment]
Thank you, @craigcook!

Curious about the word "Browser" in the "Firefox Browser Developer Edition" headings. Also, seems there were a lot of changes to the Firefox pages this week, and DevEdition is no longer in the top navigation menu under "Browsers." Do you know the reasoning behind that change?

**Comment by craigcook** [comment]
> Curious about the word "Browser" in the "Firefox Browser Developer Edition" headings. 

This is the official brand name as we move to "Firefox" being the master brand of a family of products. Firefox-the-browser is Firefox Browser. Other channels are Firefox Browser Beta, Firefox Browser Nightly, etc. I think naming scheme is still evolving so it's possible this could change again, but for now this is the official name.

> Also, seems there were a lot of changes to the Firefox pages this week, and DevEdition is no longer in the top navigation menu under "Browsers." Do you know the reasoning behind that change?

That's the new nav for Firefox product pages. I'm not sure why Dev wasn't included in that menu. I'll ask.
